### PR TITLE
Testing/discovery pylint

### PR DESCRIPTION
**Pull request by kspeer825**
# Description of change
Added a database focused discovery test.
Made changes in existing tests to satisfy pylint.
Running pylint in circle against tests.

# QA steps
 - [ ] automated tests passing
 - [ ] manual qa steps passing (list below)
 
# Risks

# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/db_utils.py [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+def ensure_environment_variables_set():
+    missing_envs = [x for x in [os.getenv('TAP_POSTGRES_HOST'),
+                                os.getenv('TAP_POSTGRES_USER'),
+                                os.getenv('TAP_POSTGRES_PASSWORD'),
+                                os.getenv('TAP_POSTGRES_PORT'),
+                                os.getenv('TAP_POSTGRES_DBNAME')] if x is None]
+    if len(missing_envs) != 0:
+        #pylint: disable=line-too-long
+        raise Exception("set TAP_POSTGRES_HOST, TAP_POSTGRES_DBNAME, TAP_POSTGRES_USER, TAP_POSTGRES_PASSWORD, TAP_POSTGRES_PORT")
```
Comment (asaf-erlich) [inline tests/db_utils.py:13]: Not a huge deal, but you can replace this message with a format string of "set {missing_envs}".format(missing_envs=missing_envs).

File: tests/test_postgres_discovery.py [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+    def get_properties(self, original_properties=True):
+        return_value = {
+            'host' : os.getenv('TAP_POSTGRES_HOST'),
+            'dbname' : os.getenv('TAP_POSTGRES_DBNAME'),
+            'port' : os.getenv('TAP_POSTGRES_PORT'),
+            'user' : os.getenv('TAP_POSTGRES_USER'),
+            'default_replication_method' : self.FULL_TABLE,
+            'filter_dbs' : 'discovery1'
+            # 'logical_poll_total_seconds': '10',
+            # 'wal2json_message_format': '1'
```
Comment (asaf-erlich) [inline tests/test_postgres_discovery.py:296]: Should these have TODO's or should they be removed?

**Review by asaf-erlich — APPROVED** [review]
approved with minor comments

File: tests/db_utils.py (hunk 1 above)
Comment (kspeer825) [reply tests/db_utils.py:13]: mmm yeah that would be better. I'll rework this.

Comment (kspeer825) [reply tests/db_utils.py]: Updated this, and calling it in all tests now.

File: tests/test_postgres_discovery.py (hunk 2 above)
Comment (kspeer825) [reply tests/test_postgres_discovery.py]: Removed the poll. Set wal2json with a TODO

**Pull request merged**